src/task_11_1.py

```diff
@@ -0,0 +1,215 @@
+"""
+    Создамем пять классов с 3-я приватными атрибутами
+    и миниму двумя методами(дданные методы будут getter, setter
+    и конструктор
+"""
+
+
+class car:
+    # Конструктор
+    def __init__(self, name: str, year: int, price: float):
+        self.__name = name
+        self.__year = year
+        self.__price = price
+
+    # getter он является методам класса
+    @property
+    def name(self):
+        return self.__name
+
+    # setter он является методам класса
+    @name.setter
+    def name(self, name):
+        self.__name = name
+
+    @property
+    def year(self):
+        return self.__year
+
+    @year.setter
+    def year(self, year: int):
+        self.__year = year
+
+    @property
+    def price(self):
```

Comment:
Please add typing everywhere
```suggestion
    def price(self) -> float:
```